src/components/categories/Category.css

```diff
@@ -13,20 +13,14 @@
 }
 
 .categories {
-    display: flex;
-    flex-flow: column nowrap;
-    justify-content: center;
-    justify-items: center;
-    max-width: 25%;
-    margin: auto;
+    text-align: center;
 }
-.category-item {
+
+.category{
     display: flex;
-    flex-flow: column nowrap;
-    justify-content: center;
-    margin: 0.5rem 0.5rem;
-    padding: 0.35rem 1.5rem;
-    box-shadow: 0.15rem 0.15rem 0.3rem #90969b;
+    align-items: center;
+    justify-content: left;
+    margin-top: 10px;
 }
```

Comment:
if you want to be over-the-top about it, this makes that table effect
otherwise, approved anyway
```
margin-top: 0px;

padding-right: 300px;

border: solid black 1px;
```